cadc-util/src/main/java/ca/nrc/cadc/log/WebServiceLogInfo.java

```diff
@@ -211,6 +215,9 @@ private void populate(StringBuilder sb, Class<?> c) {
                         } 
                         sb.append("\"").append(f.getName()).append("\"");
                         sb.append(":");
+                        if (o instanceof URI) {
+                            o = ((URI) o).toASCIIString();
+                        }
                         if (o instanceof List<?>) {
```

Comment:
`else if` to match the pattern?

Reply:
no, it's turning the value into a String so the next if/else will output as string (with quotes), so it has to be separate. If there were 2+ value types to convert, I would have put a method to make it more clear what's happening.

I'll add a comment.